.github/workflows/quiz-manager.yml

```
@@ -0,0 +1,25 @@
+
+name: quiz-manager
+on: [pull_request]
+jobs:
+  my_review:
+    name: quiz-manager review
```

Give this job a more suggestive name. 
```suggestion
  syntax_checker:
    name: Human Readable Syntax Checker
```